provision/provision-site.sh

```diff
@@ -120,7 +125,7 @@ function vvv_process_site_hosts() {
     fi
   else
     echo " * Adding hosts from the VVV config entry"
-    for line in $(shyaml get-values "sites.${SITE_ESCAPED}.hosts" 2> /dev/null < ${VVV_CONFIG}); do
+    for line in $(get_hosts_list); do
```

Comment:
shouldn't it be looping over `hosts` instead of `get_hosts_list`?

Reply:
`get_hosts_list` is a new function i added which does precisely this. Created a function to reduce code duplication slightly.

`get_hosts` does the same, but puts this in a single line, separated by space, to be used on the nginx conf.

Reply:
I get that, it's more that you used the function several lines up and stored the result in a variable, we might avoid duplicated calls

Reply:
🤦 Fixed